### PR TITLE
Authn decoupling

### PR DESCRIPTION
**Pull request by poquirion**

### CODE REVIEW

**Comment by poquirion** [comment]
It looks good to me. Maybe we could add a configuration for the candig-server port, but having 80 for it inside the container network is not problematic.

Will we need and update on the [integration test suit](https://github.com/CanDIG/candig-server/blob/develop/tests/integration/test_authflows.py) once this is put on place?

File: template/compose.tpl/containers_network.yml.tpl [hunk 1]
```diff
@@ -53,6 +53,8 @@ services:
     - tyk
   candig_server:
     image: c3genomics/candig_server
+    ports:
+    - "80:80"
```
Comment (poquirion) [inline template/compose.tpl/containers_network.yml.tpl]: Why are we exposing the port outside of the compose network?

Comment (poquirion) [inline template/compose.tpl/containers_network.yml.tpl]: Why are we exposing the port outside of the compose network?

Comment (amanjeev) [reply template/compose.tpl/containers_network.yml.tpl]: Yea this is my bad. I will remove this and push. Good catch!

**Comment by amanjeev** [comment]
I have added @haoyuanli as he is doing the candig-server merge.

File: template/compose.tpl/containers_network.yml.tpl (hunk 1 above)
Comment (poquirion) [reply template/compose.tpl/containers_network.yml.tpl]: Just removed it. I will merge all to devel and test it

**Comment by amanjeev** [comment]
Sorry, I had a call with Jonathan so was busy there. Thank you!

**Pull request merged**